### PR TITLE
Feature digest improvements

### PR DESCRIPTION
**Pull request by gorzelinski**
### Suggested merge commit message ([convention](https://ckeditor.com/docs/ckeditor5/latest/framework/contributing/git-commit-message-convention.html))

Docs: Feature digest improvements.

---

### Additional information

Closes: https://github.com/ckeditor/ckeditor5/issues/17495

### CODE REVIEW

**Comment by gorzelinski** [comment]
I fixed the things pointed out during the TP. I also added some CSS styles to the borders. Now, they group features and subfeatures. The groups are more visible, but does it look better? I dunno. You can check for yourself.

File: docs/_snippets/features/feature-digest.js
Comment (gorzelinski) [inline docs/_snippets/features/feature-digest.js:1]: Unfortunately, this empty file is necessary.

**Review by pszczesniak — COMMENTED** [review]
I'm not convinced how those rows = features + sub-features are colored. Maybe we could colored them alternately: feature with all sub-features with one colour, next with other and so on.

Pasting example to show you what is on my mind - but it would be good to discuss it, WDYT?

File: docs/_snippets/features/feature-digest.html [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+	tr.feature > td:first-child {
+		border-top-color: #000;
+		border-left-color: #000;
+	}
+
+	tr.feature > td:last-child {
+		border-top-color: #000;
+		border-right-color: #000;
+	}
+
```
[the remark this answers is not in the document]
Comment (gorzelinski) [reply docs/_snippets/features/feature-digest.html:47]: Yeah, it works, so I committed it. It's a shame we must do such gymnastics to make it work.

**Review by pszczesniak — APPROVED** [review]

**Pull request merged**